src/linalg/LinAlg.jl

```diff
@@ -1,6 +1,54 @@
 module LinAlg
-include("./graphmatrices.jl")
-end
 
+using SimpleTraits
+using ..LightGraphs
+
+import LightGraphs: IsDirected, adjacency_matrix, laplacian_matrix, laplacian_spectrum
```

Comment:
Use `.LightGraphs` here to refer to the module you just imported with `using ..LightGraphs`

Reply:
That resulted in `ERROR: LoadError: LoadError: in module path: LightGraphs not defined`

Reply:
You did `import .LightGraphs: IsDirected, ...`?

:/ I guess never mind. It's weird to me that that didn't work.

Reply:
> You did import .LightGraphs: IsDirected, ...?

Precisely that.